docs/sources/alerting/unified-alerting/high-availability.md

```diff
@@ -0,0 +1,64 @@
+
+When it comes to a highly available setup, the operational mode of the scheduler is unaffected. All alerts will be evaluated on every instance. The operational change happens in the Alertmanager, it is in charge of also **deduplicating** the alert notifications between Grafana instances.
+
+```
+  .─────.
+ ╱       ╲                                                                      ┌────────────────┐
+(  User   )──────┐                        ┌──────────────────────────────────┐  │                │
+ `.     ,'       │                        │┌─────────┐      ┌──────────────┐ │  │                ▼
+   `───'         │                        ││Scheduler│──────▶Alertmananager│─┼──┘    ┌──────────────────────┐
+                 │      ┌───────────┐  ┌─▶│└─────────┘      ▲──────────────┤ │       │                      │
+  .─────.        │      │   Load    │  │  │Grafana          │              │ │       │                      │
+ ╱       ╲       │      │ Balancing │  │  └─────────────────┼──────────────┼─┘       │     Integrations     │
+(  User   )──────┼─────▶│  Reverse  │──┤  ┌─────────────────┼──────────────┼─┐       │                      │
+ `.     ,'       │      │   Proxy   │  │  │┌─────────┐      ├──────────────▼ │       │                      │
+   `───'         │      └───────────┘  │  ││Scheduler│──────▶Alertmananager│─┼──┐    └──────────────────────┘
+                 │                     └─▶│└─────────┘      └──────────────┘ │  │                ▲
+  .─────.        │                        │Grafana                           │  │                │
+ ╱       ╲       │                        └──────────────────────────────────┘  └────────────────┘
+(  User   )──────┘
+ `.     ,'
+   `───'
+```
+
+The coordination between Grafana instances happens via [Gossip](https://en.wikipedia.org/wiki/Gossip_protocol). Alerts are not gossiped between instances. It is expected that each scheduler delivers the same alerts to each Alertmanager.
+
+The two type of messages that are gossiped between instances are:
+
+- Notification logs: Who (which instance) notifier what (which alert)
+- Silences: If an alert should fire or not
+
+These two states are persisted to the database periodically and when Grafana is successfully shutdown.
+
+## Setup
+
+To enable high availability support, we need to set the configuration [option `ha_peers` within the `[unified_alerting]` section]({{< relref "../../administration/configuration.md#unified_alerting" >}}). Setting this option to at least 1 peer will trigger the high availability mode for Grafana 8 Alerts. Communication happens using the port 9094 on both UDP and TCP, so please make sure each instance has access on these.
```

Comment:
"we" sounds a bit strange in documentation.

Reply:
Thanks for spotting this! It's a mistake.

You're right. Technical documentation should be written in an imperative mood and "we" does not align with that.